### PR TITLE
Add ActiveMQ extension

### LINKED ISSUE #424: Add support for camel-activemq
These are heavily used in integrations.

### PR DESCRIPTION
**Pull request by jamesnetherton**
Fixes #424

### CODE REVIEW

**Comment by lburgazzoli** [comment]
Wonder if it could make sense to put this among the messaging it

**Comment by jamesnetherton** [comment]
Messaging has JMS 2 on the classpath so it potentially leads to false positives when testing. Hence I chose to separate into its own itest module.

File: extensions/activemq/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# limitations under the License.
+#
+
+---
+name: "Camel Quarkus ActiveMQ"
+description: "Camel Quarkus ActiveMQ Support"
+metadata:
+  keywords:
+    - "camel"
+    - "ActiveMQ"
```
Comment (ppalaga) [inline extensions/activemq/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: I think used lower case quite consequently so far.

Comment (jamesnetherton) [reply extensions/activemq/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: I'll change it in a follow up commit. AFAIK, the keyword search on code.quarkus.io is case insensitive, so it'll still work fine with a mixture of upper & lower case.

**Pull request merged**